### PR TITLE
Pass offset instead of timeZoneName to IntlDateFormatter

### PR DESCRIPTION
**Pull request by dlondero**
Addressing  https://github.com/cargomedia/sk/issues/3861

### CODE REVIEW

**Comment by dlondero** [comment]
@njam please review

File: library/CM/Model/Location.php [hunk 1]
```diff
@@ -125,6 +125,9 @@ public function getTimeZone() {
             return null;
         }
         $timezoneNameList = DateTimeZone::listIdentifiers();
+        $timezoneNameList = \Functional\select($timezoneNameList, function ($timeZone) {
```
Comment (njam) [inline library/CM/Model/Location.php]: How about making this more explicit (and not depend on an implicit bool cast)?
```
reject($list, fn($tz) { return null === foo($tz); })
```

Comment (dlondero) [inline library/CM/Model/Location.php]: Agree.

**Comment by dlondero** [comment]
@njam please check last commit

**Comment by njam** [comment]
lgtm

**Pull request merged**